certbot-apache/certbot_apache/configurator.py

```diff
@@ -2276,6 +2276,9 @@ def perform(self, achalls):
                 http_doer.add_chall(achall, i)
             else:  # tls-sni-01
                 sni_doer.add_chall(achall, i)
+                logger.warning("TLS-SNI-01 is deprecated, and will stop working in February 2019. "
```

Comment:
I don't have a strong preference, but I'd personally like us to remove the date here as things like that can change but a released version of Certbot cannot.